resources/frameworks.yaml

```diff
@@ -127,6 +127,12 @@ MLNet:
   version: 'latest'
   description: |
     MLNET.CLI is a automated machine learning tool implemented by ml.net.
+  params:
+    _save_artifacts: [] # supported option: logs, models
+    classifiers: 'LGBM;RF;FASTTREE;LBFGS;SDCA' # supported trainers: LGBM, RF, FASTTREE, LBFGS, SDCA
+    regressors: 'LGBM;RF;FASTTREE;LBFGS;SDCA' # supported trainers: LGBM, RF, FASTTREE, LBFGS, SDCA
+    automl_type: 'NNI' # supported type: NNI, AutoML
+    
```

Comment:
non-default params are not allowed in this file, if you need to try different configurations for your personal use, please add your own definitions in `~/.config/automlbenchmark/framework.yaml`
See https://github.com/openml/automlbenchmark/blob/master/docs/HOWTO.md#custom-configuration for details